spec/classes/dns_init_spec.rb

```diff
@@ -169,7 +178,9 @@
           "        include \"#{options_path}\";",
           '};',
           '// Public view read by Server Admin',
-          "include \"#{etc_named_directory}/zones.conf\";"
+          "include \"#{etc_named_directory}/zones.conf\";",
+          "// Include logging settings",
+          "include \"#{etc_named_directory}/logging.conf\";"
```

Comment:
Feel free to add a trailing comma
```suggestion
          "include \"#{etc_named_directory}/logging.conf\";",
```